docs/Alias/Registration.md

```diff
@@ -46,7 +43,7 @@ sequenceDiagram
 
 #### Malicious [internal user](../Stakeholders/Internal%20user.md)
 
-The reason why there can be only one alias for SSB ID is to prevent a malicious internal user from exhausting many or all possible aliases in case the room accidentally allows such malicious user to become an internal user. Arguably, some room implementations could choose to relax this choice, perhaps to allow different aliases for an internal user, that covers typographic mistakes such as `aliec`, `alicce`. For the time being, it seems sensible that each internal user can receive only one alias.
+A malicious internal user could take many or all possible aliases in case the room accidentally allows such malicious user to become an internal user. Arguably, some room implementations could choose to allow only one alias per internal user, and that would still be compliant with this spec.
```

Comment:
have you considered allowing/imlementing revocation of all aliases associated with a feedid? is this out of scope for the spec?

Reply:
Yeah, that could be done as `revokeAll()`, doesn't seem too consequential to specify at this moment, we can do it in the future, cleanly, if we find a need